models/user.js

```diff
@@ -0,0 +1,14 @@
+import mongoose from "mongoose";
+import passportLocalMongoose from "passport-local-mongoose"
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
+    email: {
+        type: String,
+        required: true
+    }
+});
+
+userSchema.plugin(passportLocalMongoose);
```

Comment:
can you add a comment for why we need this?

Reply:
done